CHANGELOG.md

```diff
@@ -5,13 +5,18 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
+- Add support for Slurm accounting in US isolated regions.
 
 **CHANGES**
+- Avoid duplication of nodes seen by ClusterManager if compute nodes are added to multiple Slurm partitions.
+- ParallelCluster AMI for US isolated regions are now vended with preconfigured CA certificates to make nodes bootstrap faster and more secure.
```

Comment:
I'd remove "and more secure".

Reply:
to make nodes bootstrap faster and more secure. --> to speed up nodes bootstrap.

Reply:
_Observation just for knowledge sharing_
I agree to remove the "more secure" statement because it may open too many questions. 
Nevertheless it is true because when AMIs do not come with preconfigured certificates, they must be downloaded with insecure connection from internal yum repos.